app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe | str) -> None | str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            Cafe(cafe).visit_cafe(friend)
```

Comment:
Code Efficiency: Here, you are creating a new instance of `Cafe` for each friend. This can be inefficient if you have a large number of friends. Consider creating the `Cafe` instance once and reusing it for each friend.